src/NzbDrone.Automation.Test/Sonarr.Automation.Test.csproj

```diff
@@ -5,6 +5,10 @@
   <ItemGroup>
     <PackageReference Include="Selenium.Support" Version="3.141.0" />
     <PackageReference Include="Selenium.WebDriver.ChromeDriver" Version="92.0.4515.10700" />
```

Comment:
I decided not to touch ChromeDriver in case you are using a specific version of google chrome in your pipelines. But I had to update it locally to be able to run autotests using the freshest chrome